File origsrc/calceph-3.5.0/doc/source/confcommon.py is a regular file while file src/calceph-3.5.0/doc/source/confcommon.py is a symbolic link
File origsrc/calceph-3.5.0/mexapi/packaging/octave/createpackageoctave.sh is a regular file while file src/calceph-3.5.0/mexapi/packaging/octave/createpackageoctave.sh is a symbolic link
File origsrc/calceph-3.5.0/mexapi/src/Constants.m is a regular file while file src/calceph-3.5.0/mexapi/src/Constants.m is a symbolic link
--- origsrc/calceph-3.5.0/pythonapi/src/Makefile.am	2019-02-05 07:14:35.000000000 +0900
+++ src/calceph-3.5.0/pythonapi/src/Makefile.am	2021-09-02 14:53:27.086200200 +0900
@@ -64,7 +64,7 @@
 pyexec_LTLIBRARIES = calcephpy.la
 
 calcephpy_la_SOURCES = calcephpy.c 
-calcephpy_la_LDFLAGS = -avoid-version -module $(PYTHON_LIBS)  -shared
+calcephpy_la_LDFLAGS = -avoid-version -module $(PYTHON_LIBS)  -shared $(CALCEPHWIN32_LDFLAGS)
 calcephpy_la_LIBADD = $(top_builddir)/src/libcalcephnoinst.la
 calcephpy_la_CPPFLAGS = $(PYTHON_CPPFLAGS) -I$(top_srcdir)/src
 
@@ -122,7 +122,7 @@ calcephpy.c: $(MAYBE_PYTHONAPI_CALCEPHPY
 
 
 calcephpy_c_target :
-	test "$(top_srcdir)" == "$(top_builddir)" || cp -p $(top_srcdir)/pythonapi/src/calcephpy.p*  $(top_builddir)/pythonapi/src/
+	test "$(top_srcdir)" == "$(top_builddir)" || cp -p $(top_srcdir)/pythonapi/src/calcephpy.pxd $(top_srcdir)/pythonapi/src/calcephpy.pyx.in $(top_builddir)/pythonapi/src/
 	CC="$(CC) -I$(top_srcdir)/src $(CFLAGS)" $(PYTHON)  setup.py build_ext -i
 	sed -i -e 's://.*::g'  $(top_builddir)/pythonapi/src/calcephpy.c
 
File origsrc/calceph-3.5.0/pythonapi/src/calcephpy.pyx is a regular file while file src/calceph-3.5.0/pythonapi/src/calcephpy.pyx is a symbolic link
File origsrc/calceph-3.5.0/src/f2003calceph.f is a regular file while file src/calceph-3.5.0/src/f2003calceph.f is a symbolic link
File origsrc/calceph-3.5.0/src/f90calceph.h is a regular file while file src/calceph-3.5.0/src/f90calceph.h is a symbolic link
